.github/workflows/agent-pr.yml

```diff
@@ -22,16 +22,25 @@ jobs:
           git config --global user.name Netdatabot
           git config --global user.email bot@netdata.cloud
       - name: Create Branch
-        run: git checkout -b agent-${{ github.event.inputs.agent_version }}
+        run: |
+          git checkout -b agent-${{ github.event.inputs.agent_version }}
+          git push -u origin agent-${{ github.event.inputs.agent_version }}
```

Comment:
Why do we need this during _Create Branch_?

Reply:
The branch needs to exist on GitHub for the commit action later to actually work. Just pushing it like this is the quickest way to ensure that.